test/common/verify_3scale_UIBBT_alerts.go

```diff
@@ -188,7 +188,7 @@ func isThreeScaleUIBBTAlertFiring(ctx *TestingContext, t TestingTB) (bool, error
 
 func scaleDeploymentsConfig(t TestingTB, name string, namespace string, replicas int32, client k8sclient.Client) error {
 	retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
-		deploymentConfig := &openshiftappsv1.DeploymentConfig{
+		deploymentConfig := &appsv1.Deployment{
```

Comment:
Same with this file (`test/common/verify_3scale_UIBBT_alerts.go`), there are still a few references to deploymentConfigs.


```suggestion
		deployment := &appsv1.Deployment{
```